contracts/RelayHub.sol

```diff
@@ -205,6 +205,9 @@ contract RelayHub is IRelayHub {
         vars.success = success;
         vars.innerGasUsed = vars.gasBeforeInner-gasleft();
         (vars.status, vars.relayedCallReturnValue) = abi.decode(relayCallStatus, (RelayCallStatus, bytes));
+        if ( vars.relayedCallReturnValue.length>0 ) {
+            emit TransactionResult(vars.status, vars.relayedCallReturnValue );
+        }
```

Comment:
@drortirosh, please, get your code style together. I know solhint does not care about that, I know it is nitpicking, but these spaces! 
Use js standard when in doubt)